python/triton/language/math.py

```diff
@@ -52,52 +41,6 @@ def byte_perm(arg0, arg1, arg2, _builder=None):
     }, is_pure=True, _builder=_builder)
 
 
-@core.extern
-def min(arg0, arg1, propagate_nan: core.constexpr = PropagateNan.NONE, _builder=None):
```

Comment:
this will break backward compatibility right? Are we fine with that? (@ptillet)

Reply:
Yeah, any client code relying on calling this directly would need to be updated. FWIW, as far as I can tell, it should NOT be called directly, unless as a hack.

Reply:
Yeah. Maybe it's time to move important math functions into our standard library and move back libdevice functions into `nvidia.extra`